### DISCUSSION TITLE
Google auth login is not working

### DISCUSSION THREAD

**Discussion by sunlight7777777** [community/community#109610] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Google auth login is not working with react 18.
I have to implement the google SSO using react 18.
Please help me.
Thanks.

**Comment by 77v7777** [comment, marked as answer]
React 18 is not supported this Library version.

So for solving this problem, please use this command.

`npm install react-google-login --legacy-peer-deps`

**Comment by ninjarangja** [comment]
pls do support react 18 i'm wating for this
